eva_submission/eload_ingestion.py

```diff
@@ -337,6 +339,7 @@ def run_variant_load_workflow(self, vcf_files_to_ingest, annotation_only, resume
             'valid_vcfs': vcf_files_to_ingest,
             'vep_path': cfg['vep_path'],
             'load_job_props': job_props,
+            'acc_import_job_props': {'db.collections.variants.name': 'variants_2_0'},
```

Comment:
This looks a bit odd, but I guess we will probably change this section of the code anyway once we migrate to using the `SpringPropertiesGenerator` everywhere...

Reply:
Yes, I agree. 
I don't like the way we're passing properties to the variant load, but didn't want to change it all now.